Core/LocalAdmin.cs

```diff
@@ -594,14 +640,22 @@ private void StartSession()
 
     private static void Menu()
     {
+        var headerLines = new string[] {
+            $"SCP: Secret Laboratory - LocalAdmin v. {VersionString}",
+            string.Empty,
+            "Licensed under The MIT License (use command \"license\" to get license text).",
+            "Copyright by Łukasz \"zabszk\" Jurczyk and KernelError, 2019 - 2024",
+            string.Empty,
+            "Type 'help' to get list of available commands.",
+            string.Empty,
+        };
+
         ConsoleUtil.Clear();
-        ConsoleUtil.WriteLine($"SCP: Secret Laboratory - LocalAdmin v. {VersionString}", ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine(string.Empty, ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine("Licensed under The MIT License (use command \"license\" to get license text).", ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine("Copyright by Łukasz \"zabszk\" Jurczyk and KernelError, 2019 - 2024", ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine(string.Empty, ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine("Type 'help' to get list of available commands.", ConsoleColor.Cyan);
-        ConsoleUtil.WriteLine(string.Empty, ConsoleColor.Cyan);
+
+        foreach (var line in headerLines)
+        {
+            ConsoleUtil.WriteLine(line, ConsoleColor.Cyan, inputIntro: false);
+        }
```

Comment:
I don't see why this needs to happen as you're doing what is effectively the same job, but I would've thought it'd be better to use a verbatim string instead which looks a bit cleaner imo.

But I'll let @zabszk decide.

Reply:
In my opinion it's an unnecessary array allocation and foreach loop. You use more resources just to achieve the same effect.